project/Doc.scala

```diff
@@ -41,6 +46,8 @@ object UnidocRoot extends AutoPlugin {
 
   override def trigger = noTrigger
 
+  override def requires = BaseUnidocPlugin && ScalaUnidocPlugin && JavaUnidocPlugin
```

Comment:
`BaseUnidocPlugin` is not needed here, as both `ScalaUnidocPlugin` and `JavaUnidocPlugin` have it in their own `requires`.

Reply:
Done! Thanks, @2m.